021-quarkus-panache-multiple-pus/src/test/java/io/quarkus/qe/multiplepus/MultiplePersistenceUnitTest.java

```diff
@@ -0,0 +1,370 @@
+package io.quarkus.qe.multiplepus;
+
+import io.quarkus.qe.multiplepus.containers.MariaDbDatabaseTestResource;
+import io.quarkus.qe.multiplepus.containers.PostgreSqlDatabaseTestResource;
+import io.quarkus.qe.multiplepus.model.fruit.Fruit;
+import io.quarkus.qe.multiplepus.model.vegetable.Vegetable;
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.http.ContentType;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+import static io.restassured.RestAssured.given;
+import static io.restassured.RestAssured.when;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.hasSize;
+
+@QuarkusTest
+@QuarkusTestResource(MariaDbDatabaseTestResource.class)
+@QuarkusTestResource(PostgreSqlDatabaseTestResource.class)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
```

Comment:
I don't really like having to set a specific order to run the tests. This might prevent us to run a concrete test from the ID or using `-Dtest=MultiplePersistenceUnitTest#createInvalidPayloadFruit`. I think we can make each test to be run independiently.

Reply:
Updated!